### ISSUE TITLE
NTR: D-glyceric acid high in CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8328]
New term request D-glyceric acid high in CSF (elthat_130404085637): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
see question in https://github.com/obophenotype/human-phenotype-ontology/issues/8327

**Comment by MickeySegal** [comment]
The clinical synopsis https://www.omim.org/clinicalSynopsis/220120 lists "Increased D-glyceric acid in serum, urine, and CSF"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000372
- label: Elevated CSF D-glyceric acid concentration
- parents: [Abnormal CSF carboxylic acid concentration (HP:0500183)](https://hpo.jax.org/app/browse/term/HP:0500183)
- pmid: 
- def: The concentration of D-carboxylic acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8328
- synonyms: D-glyceric acid high in CSF(NONE)
